site/content/en/docs/start/_index.md

```diff
@@ -25,7 +25,7 @@ All you need is Docker (or similarly compatible) container or a Virtual Machine
 
 For Linux users, we provide 3 easy download options (for each architecture):
 
-### x86
+### amd64
```

Comment:
Might as well change both, if you are at it:

x86 --> amd64 (x86_64)

ARM -> arm64 (aarch64)

Do people still run 32-bit Intel ? Wow.

Reply:
Good point. New changes pushed.

Reply:
And yes, it happens sometimes. Edge case I suppose :)

Reply:
People do run 32-bit ARM (arm), but we don't advertise it. The install page would look awful, with all 5 architectures...